webservices/args.py

```diff
@@ -136,10 +136,14 @@ def make_seek_args(field=fields.Int, description=None):
 }
 
 query = {
-    'q': fields.Str(required=True, description='Text to search legal documents for.'),
+    'q': fields.Str(required=False, description='Text to search legal documents for.'),
     'from_hit': fields.Int(required=False, description='Get results starting from this index.'),
     'hits_returned': fields.Int(required=False, description='Number of results to return (max 10).'),
-    'type': fields.Str(required=False, description='Document type to refine search by')
+    'type': fields.Str(required=False, description='Document type to refine search by'),
+    'ao_no': fields.List(IStr, required=False, description='Force advisory opinion number'),
+    'ao_name': fields.List(IStr, required=False, description='Force advisory opinion name'),
+    'ao_min_date': fields.Date(descripion="Minimum issue date of advisory opinion (inclusive)"),
```

Comment:
Suggestion: for the description, earliest and latest are probably more apt than minimum and maximum. With this change, we could also get rid of the (rather technical) "(inclusive)".